src/system/game-speed.ts

```diff
@@ -3,17 +3,23 @@ import type FadeIn from "phaser3-rex-plugins/plugins/audio/fade/FadeIn";
 import type FadeOut from "phaser3-rex-plugins/plugins/audio/fade/FadeOut";
 import type BattleScene from "#app/battle-scene";
 import { globalScene } from "#app/global-scene";
-import { FixedInt } from "#app/utils";
+import { FixedNumber } from "#app/utils";
 
 type FadeIn = typeof FadeIn;
 type FadeOut = typeof FadeOut;
 
 export function initGameSpeed() {
   const thisArg = this as BattleScene;
 
-  const transformValue = (value: number | FixedInt): number => {
-    if (value instanceof FixedInt) {
-      return (value as FixedInt).value;
+  /**
+   * Adjusts the given value based on the game speed or returns it as is.
+   * @param value - The value to be evaluated. Can be either a `number` or an instance of {@linkcode FixedNumber}.
+   * @returns The original numeric value if `value` is an instance of {@linkcode FixedNumber} or if {@linkcode BattleScene.gameSpeed | gameSpeed} is `1`;
+   * otherwise, the value adjusted for the game speed.
+   */
+  const transformValue = (value: number | FixedNumber): number => {
+    if (value instanceof FixedNumber) {
+      return (value as FixedNumber).value;
```

Comment:
```suggestion
      return value.value;
```
Also might want to rename the `value` parameter so we don't have `value.value` lol.

Reply:
`num`. You may include thus in your suggestion